stock_release_channel/models/stock_release_channel.py

```diff
@@ -170,6 +170,78 @@ class StockReleaseChannel(models.Model):
     )
     last_done_picking_name = fields.Char(compute="_compute_last_done_picking")
     last_done_picking_date_done = fields.Datetime(compute="_compute_last_done_picking")
+    state = fields.Selection(
+        selection=[("open", "Open"), ("locked", "Locked"), ("asleep", "Asleep")],
+        help="The state allows you to control the availability of the release channel.\n"
+        "* Open: Manual and automatic picking assignment to the release is effective "
+        "and release operations are allowed.\n "
+        "* Locked: Release operations are forbidden. (Assignement processes are "
+        "still working)\n"
+        "* Asleep: Assigned pickings not processed are unassigned from the release "
+        "channel.\n",
+        default="open",
+    )
+    is_action_lock_allowed = fields.Boolean(
+        compute="_compute_is_action_lock_allowed",
+        help="Technical field to check if the " "action 'Lock' is allowed.",
```

Comment:
```suggestion
        help="Technical field to check if the action 'Lock' is allowed.",
```
can you cleanup such strings? Not a blocker.